copilot/README.md

```diff
@@ -0,0 +1,93 @@
+# Deploying on AWS
+
+## Introduction
+
+[Copilot](https://aws.github.io/copilot-cli/) helps simplify AWS resources and
+automate deploymnents for projects.
+
+This sample configuration runs the Open Assistant web app as an ECS Fargate
+services backed by a Serverless Aurora Postgres database.
+
+## To Setup
+
+Setup requires a few steps:
+
+```sh
+copilot app init --domain your_domain.tls
+```
+
+This will initialize and register a variety of URls with your domain.
```

Comment:
mini typo URLs

Reply:
Got it

Reply:
After doing some UI work on the web, I'll also setup copilot to launch the backend :)